stores/albert.js

```diff
@@ -0,0 +1,148 @@
+const axios = require("axios");
+const utils = require("./utils");
+
+const units = {
+    piece: { unit: "kus", factor: 1 }, // 13820x
+    kg: { unit: "kilogram", factor: 1 }, // 302x
+};
+
+exports.getCanonical = function (item, today) {
+    let quantity = item.productProposedPackaging;
+    let bio = item.badges;
+    for (let x of bio) {
+        if (x.code == "badgeattributebio") {
+            bio = true;
+            break;
+        }
+    }
+    return utils.convertUnit(
+        {
+            id: "" + item.code,
+            name: item.name,
+            // description: "", not available
+            price: item.price.value,
+            priceHistory: [{ date: today, price: item.price.value }],
+            unit: item.price.unit,
+            quantity,
+            url: item.url,
+            categoryNames: item.categoryNames,
+            ...(bio === true && { bio: true }),
+        },
+        units,
+        "albert"
+    );
+};
+
+async function sha256(message) {
+    // encode as UTF-8
+    const msgBuffer = new TextEncoder().encode(message);
+
+    // hash the message
+    const hashBuffer = await crypto.subtle.digest("SHA-256", msgBuffer);
+
+    // convert ArrayBuffer to Array
+    const hashArray = Array.from(new Uint8Array(hashBuffer));
+
+    // convert bytes to hex string
+    const hashHex = hashArray.map((b) => b.toString(16).padStart(2, "0")).join("");
+    return hashHex;
+}
+
+exports.fetchData = async function () {
+    // const fs = require("fs");
+    // return await JSON.parse(fs.readFileSync('stores/albertF.json'));
```

Comment:
to je pro ladeni, ze? To by ve vysledku nemuselo byt.

Reply:
No, není nějak jsem bojoval s update z originálu, pak mi vypadl a když jsem ho vrátil přehlídl jsem to...

Reply:
git rebase -i HEAD~4
pick/squash/squash/squash
git push --force
bez klienta děs, ale už to vypadá v jednom...

Případně ty 4 původní, kdyby náhodou:
https://github.com/eltomjan/heissepreise-cz/commit/7e360d1d902195948d0cd3c5b4f659450521eef2
https://github.com/eltomjan/heissepreise-cz/commit/17cdafe9e00af6c79304695484593005b1de0efe
https://github.com/eltomjan/heissepreise-cz/commit/ed84abb881c15ef37f8310adee33b11fe06fdfb8
https://github.com/eltomjan/heissepreise-cz/commit/dc35f01dc8eb43b38d1308624b77942630301ec6

Reply:
[Akorát to pak občas dopadne nějak takhle:<br>Push rejected, rebase or merge.<br>git push --**force**](https://miro.medium.com/v2/resize:fit:640/format:webp/0*XaLzNzYkA6PZjbl9.jpg)